index.html

```diff
@@ -0,0 +1,131 @@
+      <div class="welcome-header-text">
+        <h1>Hey there. I'm Kano</h1>
+        <h2 class="second-header">I am software developer</h2>
+      </div>
+      <div class="intro-text">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-links">
+          <li>
+            <a href="https://twitter.com/Kanombola_s">
+              <img
+                src="./assets/social-media/twitter.svg"
+                alt="twitter social media icon "
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/kanombola-kanombola-a38b061a4/"
+            >
+              <img
+                src="./assets/social-media/Linkedin.svg"
+                alt="github social media icon "
+            /></a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                src="./assets/social-media/medium.svg"
+                alt="Medium social media icon "
+            /></a>
+          </li>
+          <li>
+            <a href="https://github.com/KanoCode"
+              ><img
+                src="./assets/social-media/github.svg"
+                alt="github social media icon "
+            /></a>
+          </li>
+
+          <li>
+            <a href="#"
+              ><img
+                src="./assets/social-media/angellist.svg"
+                alt="angellist social media icon "
+            /></a>
+          </li>
+        </ul>
+      </div>
+
+      <div class="next-page">
+        <img
+          src="./assets/page-navigator.svg"
+          alt="button to go to the next page"
+        />
+      </div>
+    </section>
+    <section class="work-section">
+      <div class="section-title">
+        <!-- <h2></h2> -->
+        <span></span>
+      </div>
+      <div class="card-container">
+        <div class="card-work">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <h2 class="card-description"></h2>
+            <ul class="languages"></ul>
+            <!-- <button class="see-project"></button> -->
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="about-me-description">
+        <h2></h2>
+        <p class="intro-text"></p>
+        <!-- <button class="get-resume"></button> -->
+      </div>
+
+      <div class="card-container">
+        <div class="card-about">
+          <div class="img-placeholder"></div>
+          <div class="card-details">
+            <h2 class="card-description"></h2>
+            <ul class="languages"></ul>
+          </div>
+        </div>
+      </div>
+
+      <div class="get-in-touch">
+        <p class="intro-to-form"></p>
+        <!-- <form type action="">
+
+  
+</form> -->
```

Comment:
- [ ] Please do not add blank lines, spaces, or indentations without a reason. Kindly remove unnecessary lines and comments from your code. :+1: